hoot-js/src/main/cpp/hoot/js/conflate/js/ScriptMatchCreator.cpp

```diff
@@ -117,14 +117,6 @@ class ScriptMatchVisitor : public ConstElementVisitor
     {
       _getSearchRadius = Persistent<Function>::New(Handle<Function>::Cast(value));
     }
-
-    boost::function<bool (ConstElementPtr e)> f =
```

Comment:
Nope. The `_worstCircularError` member wasn't being used anywhere so I removed it.